src/js/jsx/tools/PolicyOverlay.jsx

```diff
@@ -64,16 +64,25 @@ define(function (require, exports, module) {
          */
         _drawDebounced: null,
 
+        /**
+         * Flag to keep track of mounted state
+         *
+         * @type {boolean}
+         */
+        _mounted: null,
```

Comment:
This makes more sense because the draw calls are debounced, and we don't otherwise have a way to cancel a debounced call. Note that lodash's version of `debounce` _does_ provide a cancellation method. So, my guess is that the aforementioned `isMounted` checks were copied and pasted from here? Furthermore, if you ever want to get fancy, you could probably also do this by adding a cancel method to our async-debounced functions.